infrastructure/postgres/store/features/create.go

```
@@ -0,0 +1,63 @@
+		p.logger.Error("failed to create feature", zap.Error(err))
+		return nil, postgres.MapError(err, "Postgres.CreateFeature")
+	}
+
+	id, err := uuid.FromBytes(m.ID.Bytes[:])
+	if err != nil {
+		p.logger.Error("failed to parse UUID from bytes", zap.Error(err))
+		return nil, postgres.MapError(err, "Postgres.ParseUUID")
+	}
+
+	config := make(map[string]any)
+	_ = json.Unmarshal(m.Config, config)
```

golangci-lint (1.64.8)

unmarshal: call of Unmarshal passes non-pointer as second argument

(govet)